targets/simple_switch/sample_extern/macsec/README.md

```diff
@@ -0,0 +1,25 @@
+# ADD CRYPTO EXTERN to SIMPLE_SWITCH
+
+For reference, the extern is implemented in crypto.cpp in this directory.
+This cpp code is to be incorporated in
+behavioral-model/targets/simple_switch/simple_switch.cpp
+
+The code is crypto.cpp is also available in the repo URL included below
+so that testing can be done using MACsec (MAC security) code.  A
+README.md is available at the repo below with steps for how to test
+the code.
+
+https://github.com/uni-tue-kn/p4-macsec
+
+The extern is defined in the following P4 file.  Search for ExternCrypt
```

Comment:
I recommend copy-pasting the P4 extern type definition here since it's only a few lines, in case the link becomes broken in the future.